airbyte-integrations/connectors/destination-elasticsearch/src/test-integration/java/io/airbyte/integrations/destination/elasticsearch/SshElasticsearchDestinationAcceptanceTest.java

```diff
@@ -0,0 +1,129 @@
+    return true;
+  }
+
+  @Override
+  protected boolean supportArrayDataTypeTest() {
+    // TODO: Enable supportArrayDataTypeTest after ticket 14568 will be done
+    return false;
+  }
+
+  @Override
+  protected boolean supportObjectDataTypeTest() {
+    return true;
+  }
+
+  @Override
+  protected TestDataComparator getTestDataComparator() {
+    return new AdvancedTestDataComparator();
+  }
+
+  private String getEndPoint() {
+    return String.format("http://%s:%d",
+        container.getContainerInfo().getNetworkSettings()
+            .getNetworks()
+            .entrySet().stream().findFirst().get().getValue().getIpAddress(),
+        container.getExposedPorts().get(0));
+  }
+
+  @Override
+  protected JsonNode getConfig() throws Exception {
+    return bastion.getTunnelConfig(getTunnelMethod(), ImmutableMap.builder().put("endpoint", getEndPoint())
+        .put("upsert", false)
+        .put("authenticationMethod", Jsons.jsonNode(ImmutableMap.builder().put("method", "basic")
+            .put("username", "elastic")
+            .put("password", ELASTIC_PASSWORD).build())));
+  }
+
+  @Override
+  protected JsonNode getFailCheckConfig() throws Exception {
+    // should result in a failed connection check
+    return bastion.getTunnelConfig(getTunnelMethod(), ImmutableMap.builder().put("endpoint", getEndPoint())
+        .put("upsert", true)
+        .put("authenticationMethod", Jsons.jsonNode(ImmutableMap.builder().put("method", "basic")
+            .put("username", "elastic")
+            .put("password", "wrongpassword").build())));
+  }
+
+  @Override
+  protected List<JsonNode> retrieveRecords(TestDestinationEnv testEnv,
+      String streamName,
+      String namespace,
+      JsonNode streamSchema)
+      throws Exception {
+    // Records returned from this method will be compared against records provided to the connector
+    // to verify they were written correctly
+    final String indexName = new ElasticsearchWriteConfig()
+        .setNamespace(namespace)
+        .setStreamName(streamName)
+        .getIndexName();
+
+    ElasticsearchConnection connection = new ElasticsearchConnection(mapper.convertValue(getConfig(), ConnectorConfiguration.class));
+    return connection.getRecords(indexName);
+  }
+
+  @Override
+  protected void setup(final TestDestinationEnv testEnv) throws Exception {
+    bastion.initAndStartBastion(network);
+    container = new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch:7.15.1")
+        .withNetwork(network)
+        .withPassword(ELASTIC_PASSWORD);
+    container.start();
+  }
+
+  @Override
+  protected void tearDown(final TestDestinationEnv testEnv) throws Exception {
+    ElasticsearchConnection connection = new ElasticsearchConnection(mapper.convertValue(getConfig(), ConnectorConfiguration.class));
+    connection.allIndices().forEach(connection::deleteIndexIfPresent);
+    connection.close();
+    container.stop();
+    container.close();
```

Comment:
Minor one. The `container.close()` is already calling `container.stop()` method under-hood so no need to call it together.